### ISSUE TITLE
Last message in the report shows for a brief moment when the keyboard opens while clicking on password text Input in password-protected pdf

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#15373]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. go to an chat
2. send a password protected pdf
3. click on  password protected pdf
4. click on enter password
5. click on password textInput

## Expected Result
last chat should not show when keyboard opens

## Actual Result
last chat flashes for a very brief moment when keyboard opens

## Workaround:
unknown 
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native

**Version Number:** 1.2.75-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/220753992-02b7f0d0-7939-46ab-84c1-53f9878eb6e9.mp4

https://user-images.githubusercontent.com/43996225/220754023-ad06ea87-35e4-41fa-8433-d962714baa78.mp4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @gadhiyamanan 
**Slack conversation:**  https://expensify.slack.com/archives/C049HHMV9SM/p1676983368278249

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @Christinadobrzyn (`Bug`), see https://stackoverflow.com/c/expensify/questions/14418 for more details.

**Comment by MelvinBot** [comment]
### Bug0 Triage Checklist ([Main S/O](https://stackoverflowteams.com/c/expensify/questions/14418))

- [ ] This "bug" occurs on a supported platform (ensure `Platforms` in OP are ✅)
- [ ] This bug is not a duplicate report (check [E/App issues](https://github.com/Expensify/App/issues) and [#expensify-bugs](https://expensify.slack.com/archives/C049HHMV9SM))
    - If it is, comment with a link to the original report, close the issue and add any novel details to the original issue instead
- [ ] This bug is reproducible using the reproduction steps in the OP. [S/O](https://stackoverflowteams.com/c/expensify/questions/14418#:~:text=The%20bug%20is%20reproducible)
    - If the reproduction steps are clear and you're unable to reproduce the bug, check with the reporter and QA first, then close the issue.
    - If the reproduction steps aren't clear and you determine the correct steps, please update the OP.
- [ ] This issue is filled out as thoroughly and clearly as possible
    - Pay special attention to the title, results, platforms where the bug occurs, and if the bug happens on staging/production.
- [ ] I have reviewed and subscribed to the linked Slack conversation to ensure Slack/Github stay in sync

**Comment by MelvinBot** [comment]
Triggered auto assignment to @ctkochan22 (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by Christinadobrzyn** [comment]
I'm not sure if this is something we want to fix now or put on hold as it's a pretty minor flicker and might be fixed with something else so going on right now (although I can't find any GHs that might relate). Adding `Engineering` label for a second opinion.

**Comment by ctkochan22** [comment]
@kavimuru 

Can you confirm that this is only on android?

**Comment by Christinadobrzyn** [comment]
Not to step in, but I could only reproduce this on Android/Chrome. 

I've tried downloading the Android Beta URL into BrowserStack, but I [can't get it to work](https://expensify.slack.com/archives/C01SKUP7QR0/p1677469587827079?thread_ts=1677210803.701409&cid=C01SKUP7QR0), so I'm unsure about Android/Native.

**Comment by ctkochan22** [comment]
@Christinadobrzyn could you do me a huge favor and get this re-assigned? 

For next steps, I'd highly recommend seeing if this is an issue with the keyboard itself. And if there are other instances or past instances. It seems strange that it would happen here.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @aldo-expensify (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by aldo-expensify** [comment]
I reproduced this in Android native, didn't reproduce anywhere else.

hmmm, my theory: this is happening because the keyboard slides up taking some time, meanwhile, I'm guessing we adjust the pdf modal's height instantly, so there is a brief moment where there is a space between the pdf modal and the keyboard that is opening. This space allows you to see the screen behind, the chat.

I does look a bit weird, so I'll make it `External` in case a contributor finds an easy solution.

**Comment by MelvinBot** [comment]
Job added to Upwork: https://www.upwork.com/jobs/~01b4d787358ad1fd5f

**Comment by MelvinBot** [comment]
Current assignee @Christinadobrzyn is eligible for the External assigner, not assigning anyone new.

**Comment by MelvinBot** [comment]
Triggered auto assignment to Contributor-plus team member for initial proposal review - @mananjadhav (`External`)

**Comment by MelvinBot** [comment]
Current assignee @aldo-expensify is eligible for the External assigner, not assigning anyone new.

**Comment by allroundexperts** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Modal does not fully cover the screen when keyboard opens.

### What is the root cause of that problem?
The root cause of this issue is setting `statusBarTranslucent` prop of the `Modal` in `AttachmentModal` to `false` [here](https://github.com/allroundexperts/Expensify/blob/7d6826b9728256380ba4d1ad0abd41dc218803cf/src/components/AttachmentModal.js#L239).

### What changes do you think we should make in order to solve the problem?

We can set `statusBarTranslucent` as `true` in all models where it is set as `false`. Doing this fixes this issue. However, it causes another [issue](https://github.com/Expensify/App/issues/9280). To fix this new issue, we can add a gap in the `Modal` content. This is currently being done for `mobile web` using the `HeaderGap` component. We can use a similar component on `android` as well.

### What alternative solutions did you explore? (Optional)
None

**Comment by tienifr** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
When we type in the password to open a password-protected PDF, the content behind the modal is briefly visible.

### What is the root cause of that problem?
This happens due to a bug fix being done incorrectly.

The `statusBarTranslucent` when `false` makes the modal does not cover the screen completely when the keyboard is opening, causing the brief flicker where we can see the content behind it.

It's set to false only for the `AttachmentModal` due to [this PR](https://github.com/Expensify/App/pull/9312). The explanation for that is [here](https://github.com/Expensify/App/pull/9210#issuecomment-1146052228).  Specifically `we don't want the modal to go over the status bar` in the `AttachmentModal`.

That fix is not fixing the root cause of the styling issue `modal will go over the status bar`. The root cause of it is that we're making the modal full screen, overlapping the status bar when `statusBarTranslucent` is true (in https://github.com/Expensify/App/blob/d5f4caa0aae0a3ab3e6242a810ffc4ec620f4453/src/components/Modal/BaseModal.js#L110) but we're not adding a proper `safe area padding` for our content. So the content can overlap with the status bar.

### What changes do you think we should make in order to solve the problem?

We need to do 2 things:
1. Remove `statusBarTranslucent={false}` in `AvatarCropModal` and `AttachmentModal` so that it will get the default `true` value, then the flickering issue here will be fixed.
2. Calculate the paddingTop of the modal correctly based on the `statusBarTranslucent` so we don't reintroduce this [ style issue ](https://github.com/Expensify/App/issues/9280). 
We can do this by fixing the `paddingTop` calculation logic in https://github.com/Expensify/App/blob/daf163e04435e293f3f7290edea86c76877e18ee/src/styles/StyleUtils.js#L317

We can update this to something like
```
let paddingTop = shouldAddTopSafeAreaPadding
        ? (modalContainerStylePaddingTop || 0) + safeAreaPaddingTop
        : modalContainerStylePaddingTop || 0;
        
    if (statusBarTranslucent) {
        paddingTop += StatusBar.currentHeight;
    }
```
(a.k.a we add `StatusBar.currentHeight` to `paddingTop` so that our content never overlaps with the status bar, we're doing the same thing for the safe area in iOS. If `statusBarTranslucent` is false we don't need this since the modal already excludes the status bar in this case.)

### What alternative solutions did you explore? (Optional)
NA



### Result
Working well after the fix:

https://user-images.githubusercontent.com/113963320/222473347-09d65c84-c1cd-48d1-a4fa-7f4bc80ec2cb.mov

**Comment by aldo-expensify** [comment]
👋  @mananjadhav do you have some thoughts on the proposals?

**Comment by mananjadhav** [comment]
Sorry I missed these. I will review these proposals today.

**Comment by situchan** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Modal height is resized earlier when keyboard opens, which makes bottom part transparent and previous screen partially shows

### What is the root cause of that problem?
The root cause is well explained by @tienifr [here](https://github.com/Expensify/App/issues/15373#issuecomment-1452061630).
So if `statusBarTranslucent` set to false in `Modal`, it handles keyboard avoiding automatically and therefore screen is already resized before fully opening keyboard.

### What changes do you think we should make in order to solve the problem?
We can completely remove conditional `statusBarTranslucent` prop in `Modal` and pass this value `true` always.
After making status bar translucent, we should handle top safe area padding in android similar to iOS on these modals:
- AttachmentModal
- AvatarCropModal

Here's final solution:
1. remove android specific [getModalStyles](https://github.com/Expensify/App/blob/main/src/styles/getModalStyles/index.android.js) file on which `shouldAddTopSafeAreaMargin` and `shouldAddTopSafeAreaPadding` was always set to `false`. Now these margin/paddings are optionally needed in android

2. In `BaseModal`, completely remove `statusBarTranslucent` prop and update relevant codes accordingly
- define `const statusBarHeight = StatusBar.currentHeight || 0;` globally
- update this line to `deviceHeight={this.props.windowHeight + statusBarHeight}`
https://github.com/Expensify/App/blob/7e8e8a1d881914abe7c13768eb13e4930cb8546b/src/components/Modal/BaseModal.js#L125
- pass `statusBarTranslucent` value always `true`
https://github.com/Expensify/App/blob/7e8e8a1d881914abe7c13768eb13e4930cb8546b/src/components/Modal/BaseModal.js#L133
- add more padding of android status bar height to `safeAreaPaddingTop` (this is the key to fix https://github.com/Expensify/App/issues/9280)
https://github.com/Expensify/App/blob/7e8e8a1d881914abe7c13768eb13e4930cb8546b/src/components/Modal/BaseModal.js#L145
to
`safeAreaPaddingTop: safeAreaPaddingTop + statusBarHeight,`
- remove `statusBarTranslucent` in [modalProptypes](https://github.com/Expensify/App/blob/main/src/components/Modal/modalPropTypes.js)

3. remove `statusBarTranslucent` prop in `AttachmentModal` and `AvatarCropModal` which are no longer needed

**Comment by tienifr** [comment]
@situchan your approach is basically the same as mine. My proposal is supposed to illustrate the approach, not intended as a whole PR ready to be merged so yes, some tweaks here and there are needed to make sure there's no regression.

**Comment by mananjadhav** [comment]
Looks like I am already running a good amount of backlog, so I'll unassign myself here. @Christinadobrzyn @aldo-expensify

**Comment by mananjadhav** [comment]
Ohh Yes! I remember. I think @aimane-chnaif would be a good C+ for this one.

**Comment by aldo-expensify** [comment]
@mananjadhav @aimane-chnaif thanks, I'll assign @aimane-chnaif then!

**Comment by MelvinBot** [comment]
📣 @aimane-chnaif You have been assigned to this job by @aldo-expensify!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by aldo-expensify** [comment]
@aimane-chnaif assigned as the C+

**Comment by MelvinBot** [comment]
@Christinadobrzyn @aldo-expensify @aimane-chnaif this issue was created 2 weeks ago. Are we close to approving a proposal? If not, what's blocking us from getting this issue assigned? Don't hesitate to create a thread in #expensify-open-source to align faster in real time. Thanks!

**Comment by conorpendergrast** [comment]
Christina is OoO, so I'm assigning myself to keep this moving!

**Comment by conorpendergrast** [comment]
Sorry, I started too early on this and thought that a proposal had been accepted! I've hired @aimane-chnaif as C+ already, my bad. 

Sorry for adding confusion here, I'll do nothing until a proposal is accepted.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by tienifr** [comment]
> We should use a new flag like `shouldAddStatusBarPaddingTop` and enable it on full-screen modals instead, and use it to decide if we want to add the status bar padding top to the modal in https://github.com/Expensify/App/blob/daf163e04435e293f3f7290edea86c76877e18ee/src/styles/StyleUtils.js#L317.

@aimane-chnaif I've explained my approach above, here's my branch for you to test out https://github.com/tienifr/App/pull/26

Please let me know if you need any other info.

Thanks!

**Comment by tienifr** [comment]
@aimane-chnaif so this would be the new platform values table (notice the new StatusBarPaddingTop, StatusBarMarginTop columns)

Platform | isSmallScreenWidth | margin:20 | StatusBarPaddingTop | StatusBarMarginTop | topSafeAreaMargin | bottomSafeAreaMargin | topSafeAreaPadding | bottomSafeAreaPadding
-- | -- | -- | -- | -- | -- | -- | -- | --
iPhone | Yes | ❌ | ❌ | ❌ | ❌ | ❌ | ✅ | ❌
Android | Yes | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | ❌
iPad | No | ✅ | ❌ | ❌ | ✅ | ✅ | ❌ | ❌
Tablet | No | ✅ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌
Web/Desktop | No | ✅ | ❌ | ❌ | ❌ | ❌ | ❌ | ❌

**Comment by conorpendergrast** [comment]
(Now that @Christinadobrzyn is back from OoO, I'm going to unassign myself again here!)

**Comment by MelvinBot** [comment]
@Christinadobrzyn @aldo-expensify @aimane-chnaif this issue is now 3 weeks old. There is one more week left before this issue <a href='https://stackoverflowteams.com/c/expensify/questions/15233/15234#15234'>breaks WAQ</a> and will need to go internal. What needs to happen to get a PR in review this week? Please create a thread in #expensify-open-source to discuss. Thanks!

**Comment by tienifr** [comment]
@aimane-chnaif sure, I think the `SafeArea` for iOS is sort of equivalent to the `statusBarHeight` in Android, so if we want to we can reuse our existing `shouldAddTopSafeAreaMargin` and `shouldAddTopSafeAreaPadding` field for the status bar while still remaining safe.

What I suggest is:
- We make `shouldAddTopSafeAreaMargin` and `shouldAddTopSafeAreaPadding` available on Android similar to iOS (rather than always setting it to `false`)
- We define our own definition of `top inset` for Android, the top inset will be set to `statusBar` height for Android, and `insets.top` (from `react-native-safe-area-context`) for iOS
This can be done by defining a `getSafeAreaPaddingTop` util similar to [here](https://github.com/tienifr/App/blob/eb464e34ecb9ed46d488e536b45b684b7c9d0f5b/src/libs/getSafeAreaPaddingTop/index.android.js) and [here](https://github.com/tienifr/App/blob/eb464e34ecb9ed46d488e536b45b684b7c9d0f5b/src/libs/getSafeAreaPaddingTop/index.js) 

@aimane-chnaif you can check the diff for this solution from [this branch](https://github.com/tienifr/App/pull/27/files), it's quite clean and easy to understand.

**Comment by tienifr** [comment]
@aimane-chnaif sure

### Proposal updated

**Comment by tienifr** [comment]
@aimane-chnaif yeah I meant the same in the RCA, maybe my explanation was a bit confusing, I updated it to be more clear.

Thanks.

**Comment by ienjoyjs** [comment]
I'd love to contribute to work on this issue. Could I possibly ask you to start work on it? Thanks!

**Comment by MelvinBot** [comment]
📣 @ienjoyjs! 📣

Hey, it seems we don’t have your contributor details yet! You'll only have to do this once, and this is how we'll hire you on Upwork.
Please follow these steps:

1. Get the email address used to login to your Expensify account. If you don't already have an Expensify account, create one [here](https://new.expensify.com/). If you have multiple accounts (e.g. one for testing), please use your main account email.
2. Get the link to your Upwork profile. It's necessary because we only pay via Upwork. You can access it by logging in, and then clicking on your name. It'll look like [this](https://www.upwork.com/freelancers/~01ca8ad8ce36b9eceb). If you don't already have an account, sign up for one [here](https://www.upwork.com/nx/signup/?dest=home).
3. Copy the format below and paste it in a comment on this issue. Replace the placeholder text with your actual details.

![Screen Shot 2022-11-16 at 4 42 54 PM](https://user-images.githubusercontent.com/30609178/202326034-1a292e64-48fd-4fef-9f4f-7feaaeedfd9f.png)

Format:
```
Contributor details
Your Expensify account email: <REPLACE EMAIL HERE>
Upwork Profile Link: <REPLACE LINK HERE>
```

**Comment by situchan** [comment]
@aldo-expensify thanks for asking me. I already gave 👍 ❤️ to suggestions.

**Comment by situchan** [comment]
I think you tagged wrong person. you meant @tienifr?

**Comment by aldo-expensify** [comment]
> @aldo-expensify thanks for asking me. I already gave 👍 ❤️ to suggestions.

ok!, I just wanted to confirm because the emojis doesn't really say with which option you are agreeing :P

**Comment by aldo-expensify** [comment]
> I think you tagged wrong person. you meant @tienifr?

oops, you are right

**Comment by MelvinBot** [comment]
📣 @situchan You have been assigned to this job by @aldo-expensify!
Please apply to this job in Upwork and leave a comment on the Github issue letting us know when we can expect a PR to be ready for review 🧑‍💻
Keep in mind: [Code of Conduct](https://github.com/Expensify/App/blob/main/CODE_OF_CONDUCT.md) | [Contributing](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md) 📖

**Comment by aldo-expensify** [comment]
@tienifr @situchan I assigned you both. How would you prefer to split the work?

**Comment by situchan** [comment]
I agree if @tienifr can quickly raise PR.

**Comment by tienifr** [comment]
@situchan [The PR](https://github.com/Expensify/App/pull/16332) is ready, Could you help to take a look at it?

**Comment by tienifr** [comment]
sorry @aimane-chnaif I'm working on the PR changes right now. I must have missed out some notifications.

**Comment by aldo-expensify** [comment]
PR merged

**Comment by aldo-expensify** [comment]
@Christinadobrzyn , for clarity:

- @aimane-chnaif is the C+
- @tienifr and @situchan are going to share the compensation

@tienifr and @situchan , about the speed bonus (if there is?), do you agree with also splitting in half?

**Comment by situchan** [comment]
I agree

**Comment by Christinadobrzyn** [comment]
OK - Hired in [upwork](https://www.upwork.com/ab/applicants/1631047568284430336/job-details)

- @aimane-chnaif $4000 as C+
- @tienifr $2000 shared comp for fix
- @situchan $2000 shared comp for fix
- @gadhiyamanan $250 as reporter

**Comment by tienifr** [comment]
Hi @aimane-chnaif @aldo-expensify @situchan 
# Regression Test Proposal

**Bug**: Last message in the report shows for a brief moment when the keyboard opens while clicking on password text Input in password-protected pdf

**Proposed Test Steps**:

Sample password protected PDF: [password-protected.pdf](https://github.com/Expensify/App/files/11043319/password-protected.pdf)
Password: 1234


1. Sign in to NewExpensify Chrome on an Android
2. Select a chat
3. Send a password protected pdf
4. Click on password protected pdf
5. Click on blue hyperlink to "enter password"
6. As the keyboard slides up, verify that you will not see the chat behind.



Do we :+1: or :-1:

**Comment by situchan** [comment]
> Sign in to NewExpensify Chrome on an Android

@tienifr I think this is wrong. It's native issue, not mWeb

**Comment by tienifr** [comment]
@situchan updated. Many thanks

**Comment by MelvinBot** [comment]
`Reviewing` label has been removed, please complete the "BugZero Checklist".

**Comment by MelvinBot** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.2.89-0 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/16332


If no regressions arise, payment will be issued on 2023-04-03. :confetti_ball:

After the hold period is over and BZ checklist items are completed, please complete any of the applicable payments for this issue, and check them off once done.
- [ ] External issue reporter
- [ ] Contributor that fixed the issue
- [ ] Contributor+ that helped on the issue and/or PR

As a reminder, here are the bonuses/penalties that should be applied for any External issue:
- Merged PR within 3 business days of assignment - 50% **bonus**
- Merged PR more than 9 business days after assignment - 50% **penalty**

**Comment by MelvinBot** [comment]
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@aimane-chnaif / @aldo-expensify] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@aimane-chnaif / @aldo-expensify] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@aimane-chnaif / @aldo-expensify] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@Christinadobrzyn] Determine if we should create a regression test for this bug.
- [ ] [@aimane-chnaif] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@Christinadobrzyn] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by Christinadobrzyn** [comment]
Paid this out! Looks like a speed bonus applies

- @aimane-chnaif $4000 + 50% bonus for C+
- @tienifr $2000 + 50% bonus shared comp for fix
- @situchan $2000 + 50% bonus shared comp for fix
- @gadhiyamanan $250 as reporter

closed the job in upwork

@aimane-chnaif and @aldo-expensify can you give your thoughts on the test regression proposal [here](https://github.com/Expensify/App/issues/15373#issuecomment-1480721110)? Thanks!

**Comment by aldo-expensify** [comment]
@aimane-chnaif / @tienifr  do you know which change introduced this bug exactly?

**Comment by situchan** [comment]
> It seems that this PR caused a regression https://expensify.slack.com/archives/C049HHMV9SM/p1680745076761309

Oh really? Didn't imagine our fix affected storybook.

I [commented](https://github.com/Expensify/App/issues/17092#issuecomment-1499509129) on that regression issue.

**Comment by MelvinBot** [comment]
@Christinadobrzyn, @aldo-expensify, @aimane-chnaif, @situchan, @tienifr Uh oh! This issue is overdue by 2 days. Don't forget to update your issues!

**Comment by MelvinBot** [comment]
@Christinadobrzyn, @aldo-expensify, @aimane-chnaif, @situchan, @tienifr Whoops! This issue is 2 days overdue. Let's get this updated quick!

**Comment by situchan** [comment]
I confirmed that storybook [issue](https://github.com/Expensify/App/issues/17092) existed before our [PR](https://github.com/Expensify/App/pull/16332) so not a regression.
I would say https://github.com/Expensify/App/pull/13580 is only the offending PR which caused regression.

**Comment by Christinadobrzyn** [comment]
Hi @situchan sorry I'm not quite sure where we are at this point, can you provide an update? Thanks!

**Comment by situchan** [comment]
> Hi @situchan sorry I'm not quite sure where we are at this point, can you provide an update? Thanks!

We can go as normal. Our PR didn't cause any regression.
I think BZ checklist is just remaining and then close the issue.

**Comment by Christinadobrzyn** [comment]
Thanks @situchan! 

@aldo-expensify and @aimane-chnaif are we happy with the test regression here - https://github.com/Expensify/App/issues/15373#issuecomment-1480721110

**Comment by Christinadobrzyn** [comment]
Regression test [here](https://github.com/Expensify/App/issues/15373#issuecomment-1484881365) - going to close this out!
